src/dimension.c

```diff
@@ -214,37 +216,34 @@ create_range_datum(FunctionCallInfo fcinfo, DimensionSlice *slice)
 }
 
 static DimensionSlice *
-calculate_open_range_default(Dimension *dim, int64 value)
+calculate_open_range_default(const Dimension *dim, int64 value)
```

Comment:
FWIW, I am doing `const` mostly to signal that pass-by-reference objects aren't modified as a side-effect by the function. Why do you think it is important to mark pass-by-value types as `const`? I guess it might help in the local scope against accidental changes within the function, but this is not what I was going for since it is not a side effect.

Reply:
Marking values as `const` is just convenience to avoid problems, not a deal-breaker. I just think we should be consistent.